uk.co.compucorp.civicrm.hrsampledata/CRM/HRSampleData/DataImporter.php

```diff
@@ -56,13 +40,17 @@ public function getDataMapping($mappingKey, $oldValue) {
   /**
    * Imports csv file into the database.
    *
+   * @param SplFileObject $fileHandler
    */
-  public function import() {
+  public function import(SplFileObject $fileHandler = null) {
     $header = null;
 
-    if (!empty($this->fileHandler)) {
-      while (!$this->fileHandler->eof()) {
-        $row = $this->fileHandler->fgetcsv();
+    if (empty($fileHandler)) {
```

Comment:
if you remove the default null value you'll also be able to remove this if, because the type declaration on the method signature will not allow null values to be passed